app/models/katello/alternate_content_source.rb

```diff
@@ -19,9 +19,10 @@ class AlternateContentSource < Katello::Model
     belongs_to :ssl_ca_cert, inverse_of: :ssl_ca_alternate_content_sources, class_name: "Katello::ContentCredential"
     belongs_to :ssl_client_cert, inverse_of: :ssl_client_alternate_content_sources, class_name: "Katello::ContentCredential"
     belongs_to :ssl_client_key, inverse_of: :ssl_key_alternate_content_sources, class_name: "Katello::ContentCredential"
-    # We breakout ssl-* validation into a function to allow for us to set
-    # the object name correctly in complex error messages
+    # We breakout ssl-* and product validation into a function to allow for us
```

Comment:
You can remove these comments. I think the code should be self explanatory plus maintaining comments will be an overhead later if things change. Things @jlsherrill taught me when I made my first few PRs.. 😄